projects/bp-gallery/src/views/gallery/search/SearchBreadcrumbs.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { Chip } from '@mui/material';
+import { useHistory } from 'react-router-dom';
+import { History } from 'history';
+import { asSearchPath, SearchType } from './SearchView';
+import './SearchBreadcrumbs.scss';
+import EventIcon from '@mui/icons-material/Event';
+import SellIcon from '@mui/icons-material/Sell';
+import DescriptionIcon from '@mui/icons-material/Description';
+import SearchIcon from '@mui/icons-material/Search';
+import PersonIcon from '@mui/icons-material/Person';
+import LocationOnIcon from '@mui/icons-material/LocationOn';
+
+type SearchParam = { type: string; value: string };
+type SearchParams = SearchParam[];
+
+const SearchBreadcrumbs = ({ searchParams }: { searchParams: URLSearchParams }) => {
+  const history: History = useHistory();
+  const searchParamValues: SearchParams = [];
+  const iconsToTypes = (searchType: SearchType) => {
```

Comment:
```suggestion
  const iconForType = (searchType: SearchType) => {
```
It's more this way round, isn't it?